source/linux/How_to_Guides/Target/How_to_manually_flash_emmc_device.rst

```diff
@@ -0,0 +1,217 @@
+How to flash eMMC device
+========================
+
+Overview
+--------
+
+- Requires a bootable SD card with linux Kernel images. Boot the target EVM to linux prompt and check for mmc devices
+
+  .. code-block:: console
```

Comment:
Indentation levels should be a multiple of 3.

Reply:
@StaticRocket This [doc](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#code-images-and-other-content-inside-lists) says otherwise for an unordered list (2 spaces)?

> To include content such as code or an image inside a list, position the code or image directive flush with the text in the list. That is, indent three spaces for ordered lists and two spaces for unordered lists.

Reply:
That is a community guide, but I'll give you that the sphinx examples for bulleted lists use 2 spaces and the docutils ref [1] says the indentation is dictated by the next line, which would normally be 2 spaces when formatted with vim/nvim gq.

I guess this is fine

[1] https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists